### ISSUE TITLE
Compatibility issue with OMS Linux agent

### ISSUE THREAD

**Issue by jjindrich** [dotnet/core#1129]
I have problem with OMS Linux agent. When I install DotNet Core 2.0.2. on Linux server with running OMS agent, I got strange errors and reporting is lost. There is issue with OMI library - OMS agent is requiring v 1.3.2, DotNet Core is requiring 1.4.1

More info https://github.com/Microsoft/OMS-Agent-for-Linux/issues/612

**Comment by richlander** [comment]
Thanks for reporting. We'll take a look.

**Comment by Petermarcu** [comment]
@richlander are you following up with the folks from https://github.com/Microsoft/OMS-Agent-for-Linux?

**Comment by richlander** [comment]
Help me understand ... you say that .NET Core requires 1.4.1 of the OMI library. I don't recall .NET Core having that dependency. Can you elaborate on that?

**Comment by jjindrich** [comment]
Yes, I will be happy to help you. What I found is that dotnet core is adding new repository which forces to use new OMI. 
Probably not working with OMI 1.4. which is required by dotnet core (https://www.microsoft.com/net/learn/get-started/linuxubuntu) - adding sourcelist repo with new OMI
Tested on clean server: when I install Ubuntu 16.04 and deploy OMS agent, is reporting all data correctly. When I deploy DotNet Core, is omi updated to 1.4.1 version and OMS agent is not working properly. Data are missing in OMS portal.
